File: Haskell/beginner/1097.hs
-- https://judge.beecrowd.com/pt/problems/view/1097
-- Sequência IJ 3

-- Recursive
sequenceIJ :: Int -> Int -> IO ()
sequenceIJ i j
    | i > 9 = return ()
    | otherwise = do
        putStrLn $ "I=" ++ show i ++ " J=" ++ show j
        putStrLn $ "I=" ++ show i ++ " J=" ++ show (j - 1)
        putStrLn $ "I=" ++ show i ++ " J=" ++ show (j - 2)
        sequenceIJ (i + 2) (j + 2)


main = sequenceIJ 1 7

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1097.c
// https://judge.beecrowd.com/pt/problems/view/1097
// Sequência IJ 3

// Libraries
#include <stdio.h>

// Main function
int main(){
    // Variables
    int i = 1;
    int j = 7;

    // Iteration
    while(i <= 9){
        printf("I=%d J=%d\n", i, j);
        printf("I=%d J=%d\n", i, j - 1);
        printf("I=%d J=%d\n", i, j - 2);
        i += 2;
        j += 2;
    }

    // Ending code
    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra